recipes/ovs_offload/1_virt_ovs_vxlan.py

```diff
@@ -54,11 +55,9 @@ def do_pings():
 def get_vxlan_dev(host):
     vxlan_port = ctl.get_alias("vxlan_port")
     vxlan_dev = "vxlan_sys_%s" % vxlan_port
-    # in asap mlnx ofed the interface name is dummy_port.
+    # In asap mlnx ofed the interface name is dummy_port.
     vxlan_dummy = "dummy_%s" % vxlan_port
-    cmd = host.run("ip l show dev %s" % vxlan_dev)
-    out = cmd.out().strip()
-    if not out:
+    if not vxlan_dev in netifaces.interfaces():
```

Comment:
How this works? You don't specify which machine to check the interfaces.

Reply:
This makes host param redundant here.
But as mentioned I need explanation why this works. This seems to only be able to work if you actually run the test from the actual hypervisor. Which is not the case in mars were it runs tests from machine not related to the tested setup.

Reply:
Yes, it worked on both fc24 and centos when I ran tests manually since it checks the interfaces of the machine that is running on it.

Now, regarding MARS, I need to specify the name of the machine.

Reply:
I made new implementation and checked it with all enabled lnst tests on fc24 and centos 7.2. I will close this pull request and create new one.